### PR TITLE
Add nav items

### PR DESCRIPTION
**Pull request by yaacov**
## 📝 Description
Add virtual machine instance and template links

## 🎥 Demo

nav-items

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -17,7 +21,44 @@
...
+        "data-quickstart-id": "qs-nav-virtualmachineinstances",
+        "data-test-id": "virtualmachineinstances-nav-item"
+      }
+    }
+  },{
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "templates",
+      "section": "virtualization",
+      "name": "%plugin__kubevirt-plugin~Teamplates%",
```
Comment (glekner) [inline console-extensions.json:52]: Teamplates -> Templates

**Review by glekner — CHANGES_REQUESTED** [review]

File: console-extensions.json (hunk 1 above)
Comment (yaacov) [reply console-extensions.json:52]: +1 Thanks !

File: console-extensions.json [hunk 2]
```diff
@@ -17,7 +21,44 @@
...
+      }
+    }
+  },{
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "templates",
+      "section": "virtualization",
+      "name": "%plugin__kubevirt-plugin~Templates%",
+      "href": "/templates",
+      "namespaced": true,
```
Comment (metalice) [inline console-extensions.json:54]: how will this handle this kind of bug: https://bugzilla.redhat.com/show_bug.cgi?id=2047186
with current architecture? something we should consider.

Comment (yaacov) [reply console-extensions.json:54]: This is the fix for that bug.
no more special namespaces for templates, you only see the namespace you are in

**Pull request merged**